requirements.txt

```diff
@@ -28,4 +28,4 @@ pybind/pybind11@3e9dfa2866941655c56877882565e7577de6fc7b --build
 msgpack/msgpack-c@cpp-3.3.0 -DMSGPACK_BUILD_TESTS=Off
 sqlite3@3.43.2 -DCMAKE_POSITION_INDEPENDENT_CODE=On
 ROCm/composable_kernel@57cdd70b7cb14e5e3b60cd9a5f96ba8dc343763e -DCK_BUILD_JIT_LIB=On -DCMAKE_POSITION_INDEPENDENT_CODE=On
-ROCm/rocMLIR@3612396bca1139abf25e2ed0085fe481d275af89 -DBUILD_FAT_LIBROCKCOMPILER=On
+ROCm/rocMLIR@71193918bdd94e75fd92b3a6dc2ed58e74b52131 -DBUILD_FAT_LIBROCKCOMPILER=On
```

Comment:
71193918bdd94e75fd92b3a6dc2ed58e74b52131

This one is same as #3141 . @causten  this  PR can be closed